tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -235,8 +235,9 @@
 @optgroup.option(
     "--enable_chunked_context",
     is_flag=True,
-    default=False,
-    help="Enable chunking in prefill stage for enhanced throughput benchmark.",
+    default=None,
+    help="Enable chunking in prefill stage for enhanced throughput benchmark. "
+    "Default is False for PyTorch/AutoDeploy backend, True for TensorRT backend.",
 )
```

Comment:
```suggestion
@optgroup.option(
   "--chunked_context/--no_chunked_context",
    default=True,
    help="Enable/disable chunking in prefill stage for enhanced throughput benchmark. "
)
```

This will make it so that it is always set to `True` by default and still be able to be disabled. It's more explicit.

Reply:
See here for reference: https://www.perplexity.ai/search/in-python-click-how-do-you-def-X5pb0LIyRFCTeeSNwQegYg#0